### PR TITLE
Wiki

### PR DESCRIPTION
**Pull request by Araon**
I've added the wiki cog as per as issue #2 

# poc
image

and also the repo didn't have any requirments.txt so I've gone ahead and added that too

### CODE REVIEW

File: cogs/Wiki.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    try:
+        sh = wiki.summary(text, sentences = 3)
+    except:
+       sh = 'Search term is to board to summarize'
+    return sh
+
+
+class Wiki(commands.Cog):
+
+    def __inti__(self, bot):
```
Comment (Xx-Ashutosh-xX) [inline cogs/Wiki.py]:
```suggestion
    def __init__(self, bot):
```

**Review by Xx-Ashutosh-xX — CHANGES_REQUESTED** [review]
Remove the following folders :

- .vscode
- Pycache

**Comment by Araon** [comment]
I've deleted the necessary files as requested

**Review by Xx-Ashutosh-xX — APPROVED** [review]
Looks Good !!

**Comment by Araon** [comment]
I'd suggest one thing is that rather than hardcoding the token, use a .env file to store the token.

**Pull request merged**